src/emailservice/Dockerfile

```diff
@@ -28,6 +28,19 @@ WORKDIR /email_server
 # Grab packages from builder
 COPY --from=builder /usr/local/lib/python3.7/ /usr/local/lib/python3.7/
 
+# Enable/disable Stackdriver Profiler agent
+ENV ENABLE_PROFILER 1
```

Comment:
My bad, I should've explained better earlier.

Can you add this variable to Kubernetes manifests (env section) instead as commented out? We don't want Profiler to be enabled by default as it will not work locally.

You don't need to add any of the comments below, just say:

```
# Uncomment the following line to enable Stackdriver Profiler agent
ENABLE_STACKDRIVER_PROFILING: "1"
```

Users will never edit the dockerfile, some of them use images as is.

Also nobody will set their GCP_PROJECT_ID (again, remember, this project is supposed to run almost zero config), almost all Stackdriver components automatically determine project ID from GCE metadata. (If not, we must fix them). If a user of demo is not on GCP and yet they want to try Stackdriver profiler, they can set project ID in the code directly.

So let's eliminate these two env vars at the end.

Reply:
I moved the environment variable to the manifests. 

When Profiler agents are running in GCP env, there is no need to set the env vars as they will be determined from the metadata server. Agents will work in a non-GCP environment if the two env vars are set correctly. 

What is the best way to keep the changes minimal for users who want to try the agents in a non-GCP env? Changing code seems a bit of a barrier for those who want to quickly run code and play with the Profiler flame graph in the Cloud Console.

Reply:
@kalyanac Currently we don't support that scenario for Tracing either. In some services, you'll see commented out ProjectID field for profiler/tracing code. But nobody asked for it yet, so let's leave that scenario out.

For now, let's assume if user set `ENABLE_STACKDRIVER_PROFILING=1`, they're running on GCE/GKE so they don't need to set projectID/credentials.

Reply:
Sounds good. Will merge this for now